base/game_events/GameEventManager.ts

```diff
@@ -182,7 +182,7 @@ export class GameEventManager {
             voice_key: npc.voice_key,
             hero_direction: this.data.hero.current_direction,
         });
-        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async (finished: any) => {
+        this.fire_next_step = dialog_manager.next.bind(dialog_manager, async (finished: !!any) => {
```

Comment:
@jjppof let me know if this works

Reply:
No, do this: `finished: boolean`. Remove the `any`.